desk/app/grouper.hoon

```diff
@@ -104,30 +126,48 @@
   ?-  -.sign
       %poke-ack   `this
       %watch-ack  `this
-      %kick       
+      %kick
     :_  this
     ~[(bite-subscribe bowl)]
     ::
       %fact
     =+  !<(=bite:reel q.cage.sign)
-    ~?  dev-mode  [bite (~(has in enabled-groups) token.bite)]
-    ?>  (~(has in enabled-groups) token.bite)
-    ?>  ?=([%bite-1 *] bite)
```

Comment:
I don't see any changes here that stop producing `%bite-1` or start producing `%bite-2`. Am I missing something?

Also did `%bite-1` ever make it out into production-production? I thought that was your recent addition. If not, could be worth just updating the `%bite-1` case and calling it a day.

Reply:
those changes went through on the landscape side here: https://github.com/tloncorp/landscape/blob/develop/desk/app/bait.hoon#L136-L150